src/transformers/models/phi3/modeling_phi3.py

```diff
@@ -171,7 +172,7 @@ def __init__(self, dim, config, device=None):
 
     @torch.no_grad()
     def forward(self, x, position_ids, seq_len=None):
-        seq_len = torch.max(position_ids) + 1
+        seq_len = position_ids.shape[-1]
```

Comment:
Just realized that position ids are not always same size as input. Will come back to revert this later, which means that compile still doesn;t work for rope scaling in Phi3

Reply:
Correct, with left-padding the maximum value of `position_ids` is not the sequence length. For that (sequence length) we have `cache_positions` :)